.github/workflows/test_comprehensive.yml

```diff
@@ -9,6 +9,9 @@ on:
       - "v*x"
     tags:
       - "v*" # Push events to matching v*, i.e. v1.0, v20.15.10
+  pull_request:
+    paths:
+      - '.github/workflows/test_comprehensive.yml'
```

Comment:
What's this change for?

Reply:
Oh I get it you're making this proc on PR.

Reply:
I assume you will delete this after tests go green.

Reply:
Why? I think that we should ths workflow every time when someone edits it.